src/keys.ts

```diff
@@ -18,6 +18,7 @@ interface I18NextParsedOptions {
  */
 export interface ExtractedKey {
   key: string;
+  keyPrefix?: string;
```

Comment:
Would it be possible to move this option to `parsedOptions`, make it non-optional and default it to `null`?